src/layer/water.js

```diff
@@ -137,7 +137,7 @@ const labelPaintProperties = {
 const labelLayoutProperties = {
   "symbol-placement": "line",
   "text-field": Label.localizedNameInline,
-  "text-font": ["OpenHistorical Italic"],
+  "text-font": ["Americana-Italic"],
```

Comment:
There’s some interest in replacing this italic sans-serif font with an italic serif font for natural features like waterways and waterbodies. Let’s track that as tail work, since it may require some research into analogues in other writing systems.